content/cn/blog/_posts/2018-06-07-dynamic-ingress-kubernetes

```diff
@@ -232,4 +209,4 @@ If you’re interested in using Ambassador as an API Gateway or Kubernetes ingre
 
  
 
-如果您有兴趣将 Ambassador 用作 API 网关或 Kubernetes 入口解决方案，请查看[Ambassador入门指南]（https://www.getambassador.io/user-guide/getting--启动 ）。
+如果您有兴趣将 Ambassador 用作 API 网关或 Kubernetes ingress 解决方案，请查看[Ambassador入门指南]（https://www.getambassador.io/user-guide/getting）。
```

Comment:
I figured the original URL link is something like "https://www.getambassador.io/user-guide/getting-started". In your translation, the link now reports 404.